### PR TITLE
Add Gitlab token to SSM Parameter Store parameters

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request adds the Gitlab token to the list of SSM Parameter Store parameters that can be read by the AMI build user.

## 💭 Motivation and Context

The build user must be able to read the Gitlab personal authorization token in order to checkout the private `dns-profile-randomizer` repo.

## 🧪 Testing

I have successfully used this code to build a prerelease AMI.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: terraform-build-user/main.tf [hunk 1]
```diff
@@ -20,6 +20,7 @@ module "iam_user" {
     "/vnc/password",
     "/vnc/ssh/rsa_public_key",
     "/vnc/ssh/rsa_private_key",
+    "/gitlab/personal_authorization_token",
```
Comment (dav3r) [inline terraform-build-user/main.tf:23]: It's probably not the worst idea for us to alphabetize our list of `ssm_parameters` here.

Comment (jsf9k) [reply terraform-build-user/main.tf]: See commit b5d413f.

Comment (dav3r) [reply terraform-build-user/main.tf]: Merci beaucoup!

**Comment by jsf9k** [comment]
> Maybe we've already covered this, and I missed it. But why is the `dns-profile-randomizer` repository private? I'd to have a group conversation about this.

You'd have to ask the assessment folks about that.  They asked that the tool be installed in the Kali image, and they host it in a private GitLab repo.

**Pull request merged**